packages/pigeon/lib/swift_generator.dart

```diff
@@ -694,3 +710,76 @@ String _nullsafeSwiftTypeForDartType(TypeDeclaration type) {
   final String nullSafe = type.isNullable ? '?' : '';
   return '${_swiftTypeForDartType(type)}$nullSafe';
 }
+
+class _SwiftFunctionArgument {
+  _SwiftFunctionArgument({
+    required this.name,
+    this.label,
+    required this.type,
+    required this.namedType,
+  });
+
+  final String name;
+  final String? label;
+  final TypeDeclaration type;
+  final NamedType namedType;
+}
+
+class _SwiftFunctionComponents {
+  _SwiftFunctionComponents._({
+    required this.name,
+    required this.arguments,
+    required this.returnType,
+    required this.method,
+  });
+
+  factory _SwiftFunctionComponents.fromMethod(Method method) {
+    if (method.swiftFunction.isEmpty) {
+      return _SwiftFunctionComponents._(
+        name: method.name,
+        returnType: method.returnType,
+        arguments: method.arguments
+            .map((NamedType e) => _SwiftFunctionArgument(
+                  name: e.name,
+                  type: e.type,
+                  namedType: e,
+                ))
+            .toList(),
+        method: method,
+      );
+    } else {
+      final String argsCapturator =
```

Comment:
I'm not sure what "capturator" means. "Capturer"? "Extractor"?

Reply:
Fixed